lib/measure.js

```diff
@@ -0,0 +1,29 @@
+import pTimeout from '../vendor/p-timeout.js'
+import { RETRIEVE_TIMEOUT } from './constants.js'
+
+/**
+ * Performs a retrieval for given node and blob and returns measurement results.
+ *
+ * @param {string} nodeAddress
+ * @param {string} blobBase64Id
+ * @param {typeof globalThis.fetch} fetch
+ * @returns {Promise<{ retrievalSucceeded: boolean }> }
+ */
+export const measure = async (nodeAddress, blobBase64Id, fetch = globalThis.fetch) => {
+  let retrievalSucceeded = true
+  try {
+    const response = await pTimeout(
+      fetch(`${nodeAddress}/v1/blobs/${blobBase64Id}`),
+      { milliseconds: RETRIEVE_TIMEOUT }
+    )
+
+    if (!response.ok) {
+      retrievalSucceeded = false
+    }
```

Comment:
Please add two console logs:
1. One printed before making the fetch request, this log should include the URL requested.
2. Another log printing the response status code.

These logs are essential to allow us to troubleshoot retrieval results produced by Checker Apps.

Reply:
Added better logging in https://github.com/CheckerNetwork/walrus-checker/pull/12/commits/b85c54ab9b080f01e6fdc5c9eff579cc6f34a80c